Additional lazy fetch tuning

putrenkov opened the pull request:
**Link to QA issue**
QA-3961 - https://github.com/BroadleafCommerce/QA/issues/3961

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961

danielcolgrove commented:
@putrenkov I was afraid of possible "session closed" issues.  We're going to need to do some testing with these changes and for sure run the integration tests locally.  If testing looks good but the integration test fails, need to try to understand why.

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961

putrenkov committed:
else

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961

putrenkov committed:
QA-3961
Additional lazy fetch tuning
https://github.com/BroadleafCommerce/QA/issues/3961
